lib/js/components/Divider/Divider.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+	<div
+		class="divider"
+		:class="{
+			'-vertical': isVertical,
+			'-horizontal': !isVertical,
+			'-strong': prominence === PROMINENCES.STRONG,
+			'-weak': prominence === PROMINENCES.WEAK,
+		}"
+	/>
+</template>
+
+<style lang="scss" scoped>
+@import '../../../styles/settings/colors/tokens';
+
+.divider {
+	background-color: $color-neutral-divider;
+
+	&.-strong {
+		background-color: $color-neutral-divider-strong;
+	}
+
+	&.-weak {
+		background-color: $color-neutral-divider-weak;
+	}
+
+	&.-vertical {
+		height: 100%;
+		width: 1px;
+	}
+
+	&.-horizontal {
+		height: 1px;
+		width: 100%;
+	}
+}
+</style>
+
+<script lang="ts">
+import { PROMINENCES } from './Divider.consts';
+
+export default {
```

Comment:
you forgot to export it in https://github.com/bethinkpl/design-system/blob/master/lib/js/index.ts